mail.py

```diff
@@ -21,24 +22,23 @@ def check_gmail():
         current_mail_id = int(file_reader.read())
 
         if possible_new_email_id > current_mail_id:
-            latest_email_id = possible_new_email_id
-            number_of_new_mails = latest_email_id - current_mail_id
-
             file_reader.close()
-
+            
+            latest_email_id = possible_new_email_id
             save_latest_email_id(latest_email_id)
 
-            raw_email = imap.fetchs([latest_email_id])
+            senders_subjects = []
+            message = 'Shalom Adonai, irmãos. Novo(s) e-mail(s), deem uma olhada:\n'
+
+            for mail_id in range(current_mail_id + 1, latest_email_id + 1):
+                raw_email = imap.fetchs([mail_id])
 
-            mail_parser = MailWrapper(raw_email)
-            sender = mail_parser.get_sender()
+                mail_parser = MailWrapper(raw_email)
 
-            subject = mail_parser.get_subject()
+                senders_subjects.append(MailSenderSubject(mail_parser.get_sender(), mail_parser.get_subject()))
 
-            if number_of_new_mails > 1:
-                message = 'Shalom Adonai, irmãos. {} novos emails, deem uma olhada.'.format(number_of_new_mails) 
-            else:
-                message = 'Shalom Adonai, irmãos. Novo email, deem uma olhada.\n\nTítulo: {}\n\nRemetente: {}'.format(subject, sender) 
+            for ss in senders_subjects:
```

Comment:
avoid using abbreviations or simplified names for variables, it's better to be explicit about what we are doing here.
This variable is something related to an old army from europe or it's a abbreviation for sender_object ?